nodes/barcode/.eslintignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Why did you add this additional `.eslintignore`?

The root `.eslintignore` already contains node_modules, and at least for me, I never encountered a node_modules folder inside a package folder.

Reply:
The target `lint` seems to start `eslint .` in the directory of each node directory. So the root `.eslintignore` is out of scope.